### ISSUE TITLE
Nullpointer exception

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ankit4u3/jtracert#1]
```
What steps will reproduce the problem?
1. I just run my large problem with the following vm arguments
-DanalyzerOutput=sdEditRtClient -DsdEditHost=127.0.0.1 -DsdEditPort=60001
-javaagent:jTracert.jar -Xms1024k -Xmx2048k
2.
3.

What is the expected output? What do you see instead?
I saw null pointer exception followed by out of memory exception. I've 2GB
of physical ram

What version of the product are you using? On what operating system?
0.0.2

Please provide any additional information below.


```

Original issue reported on code.google.com by `arunacha...@gmail.com` on 25 Jun 2008 at 7:05

**Comment by GoogleCodeExporter** [comment]
```
sorry for not to attach the log trace.
java.lang.NullPointerException
    at
com.google.code.jtracert.traceBuilder.impl.MethodCallTraceBuilderImpl.leave(Meth
odCallTraceBuilderImpl.java:237)
    at
com.google.code.jtracert.traceBuilder.impl.MethodCallTraceBuilderImpl.exception(
MethodCallTraceBuilderImpl.java:263)
    at

then the application call traces follows.
```

Original comment by `arunacha...@gmail.com` on 25 Jun 2008 at 7:15

**Comment by GoogleCodeExporter** [comment]
```
To be fixed in 0.0.3 release
```

Original comment by `Dmitry.Bedrin` on 14 Jul 2008 at 4:39
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in SVN.

However, support for huge projects is still in development.
```

Original comment by `Dmitry.Bedrin` on 27 Aug 2008 at 1:06
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in jTracert-0.0.3 release
```

Original comment by `Dmitry.Bedrin` on 8 Sep 2008 at 9:41
- Added labels: **Component-Logic**, **OpSys-All**

**Comment by GoogleCodeExporter** [comment]
```
Got this error once gave -Xmx1024k with vm options, am using jTracert-0.0.3.jar

java.lang.NullPointerException
        at com.google.code.jtracert.traceBuilder.impl.MethodCallTraceBuilderImpl
.leave(MethodCallTraceBuilderImpl.java:259)
        at com.google.code.jtracert.traceBuilder.impl.MethodCallTraceBuilderImpl
.exception(MethodCallTraceBuilderImpl.java:285)
        at org.apache.xerces.parsers.StandardParserConfiguration.parse(StandardP
arserConfiguration.java:549)
        at org.apache.xerces.parsers.StandardParserConfiguration.parse(StandardP
arserConfiguration.java:585)
        at org.apache.xerces.parsers.XMLParser.parse(XMLParser.java:147)
        at org.apache.xerces.parsers.AbstractSAXParser.parse(AbstractSAXParser.j
ava:1148)
```

Original comment by `mv.nit...@gmail.com` on 23 Oct 2008 at 9:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `Dmitry.Bedrin` on 6 Jan 2009 at 9:47
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in SVN
```

Original comment by `Dmitry.Bedrin` on 6 Jan 2009 at 9:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in jTracert 0.0.4
```

Original comment by `Dmitry.Bedrin` on 28 Jan 2009 at 8:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `Dmitry.Bedrin` on 6 Feb 2009 at 12:39
- Changed state: **Verified**
